src/components/MainMenuBar.js

```diff
@@ -0,0 +1,108 @@
+import React, { useMemo } from "react";
+import { useSelector } from "react-redux";
+import { useHistory } from "react-router-dom";
+import { useModulesManager } from "../helpers/modules";
+import MainMenuContribution from "./generics/MainMenuContribution";
+
+function getMenus(modulesManager, key, rights, menuVariant) {
+  const menus = modulesManager.getContribs(key);
+  const menuConfig = modulesManager.getConf("fe-core", "menus", []);
+  const isMenuConfigEmpty = !menuConfig || menuConfig.length === 0;
+  
+  const unmatchedMenus = getUnmatchedMenus(menuConfig, menus, rights, modulesManager, menuVariant);
+  const menuToProcess = [...menus, ...unmatchedMenus];
+
+  let processedMenus;
+  if (!isMenuConfigEmpty) {
+    const sortedMenus = sortMenus(menuToProcess, menuConfig);
+    processedMenus = processMenu(sortedMenus);
+  } else {
+    processedMenus = processMenu(menus);
+  }
+
+  return processedMenus;
+};
```

Comment:
```
function getMenus(modulesManager, key, rights, menuVariant) {
  const menus = modulesManager.getContribs(key);
  const menuConfig = modulesManager.getConf("fe-core", "menus", []);
  
  if (menuConfig?.length) {
    const unmatchedMenus = getUnmatchedMenus(menuConfig, menus, rights, modulesManager, menuVariant);
    const menuToProcess = [...menus, ...unmatchedMenus];
    const sortedMenus = sortMenus(menuToProcess, menuConfig);
    return processMenu(sortedMenus);
  }

  return processMenu(menus);
}
```

Reply:
I applied this fix in https://github.com/openimis/openimis-fe-core_js/pull/262/commits/e024531a2a5270d2ef97ea7fe325dd6510a8fe44 that will be merged into this PR.